parseGTFS.js

```
@@ -11,9 +11,22 @@ async function fetchAndParseGTFS() {
         const response = await axios.get(process.env.GTFS_KEY, { responseType: 'arraybuffer' });
         if (response.status === 200) {
             const feed = GtfsRealtimeBindings.transit_realtime.FeedMessage.decode(new Uint8Array(response.data));
-            gtfsData = feed;
+            
+            // Validate feed data
+            const validEntities = feed.entity.filter(entity => {
+                const isValid = entity.vehicle && 
+                              entity.vehicle.position && 
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)